app/services/coin_market_cap_pro/update_market_metrics_service.rb

```diff
@@ -0,0 +1,43 @@
+module CoinMarketCapPro
+  class UpdateMarketMetricsService < Patterns::Service
+    include CoinMarketCapProHelpers
+
+    def call
+      update_metrics
+    end
+
+    private
+
+    def update_metrics
+      data = load_cmc_data
+      perform_update_metrics(data) if data.present?
+    end
+
+    def perform_update_metrics(data)
+      metrics = data.dig('quote', 'USD')
+      MarketMetric.create(
```

Comment:
Should we do a `create!` instead to throw an exception if anything goes awry?

Otherwise we could still fire the Market Metrics healthcheck even if the insertion was a failure...